lib/IDE/CodeCompletion.cpp

```diff
@@ -1734,6 +1735,10 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
     this->DotLoc = DotLoc;
   }
 
+  void setIsUnwrapped(bool value) {
```

Comment:
Nitpick: could we please call this `setIsUnwrappedOptional`?  Otherwise LGTM.

Reply:
Will do. Thanks!